app/views/mortgage_calculator/affordabilities/_next_steps_content.html.erb

```diff
@@ -13,7 +13,7 @@
     <div class="callout callout--tip">
       <span class="callout__icon" aria-hidden="true">?</span>
 
-      <h5 class="callout__heading"><%= t("affordability.next_steps.learn_more.title") %></h5>
+      <h3 class="callout__heading"><%= t("affordability.next_steps.learn_more.title") %></h3>
```

Comment:
I left the h5 here and introduced the callout__heading class in the previous piece of work because it seemed semantically correct (given h4 tags elsewhere). It's a debatable point and I'm not wedded to it but either way we don't really need both h3 and callout__heading.